### PR TITLE
Settings Page layout changes

### PR DESCRIPTION
**Pull request by infinite-persistence**
## Test
- https://kp.odysee.com

## Issue
- Closes [#6363 split settings into separate subareas with navigation](https://github.com/lbryio/lbry-desktop/issues/6363)
- Closes [#6395 Advanced settings not persisting ](https://github.com/lbryio/lbry-desktop/issues/6395)
    - "Advanced settings" page no longer exists.

## Comments
- I left the individual commits unsquashed to hopefully help the review.  Probably easiest to go through each commit one-by-one.
- For medium and small screen layout, the navigation is hidden for now.  I think this is fine since we are now showing all settings (scroll) and the total settings is quite small.  If needed, we can probably display the navigation as a Tab at the top for these layouts.

## Debt
- <strike>"Creator Settings" remained in the old style. Will update the style together in an upcoming PR for "min tips". </strike>

## Notes
https://www.notion.so/lbry/Settings-Page-Debt-d07a232a03fa469e8bae47ce2bf9e307

### CODE REVIEW

**Comment by infinite-persistence** [comment]
### 2cd691d to 2b53c74 (ready for review)
Notes:  https://www.notion.so/lbry/Settings-Page-Debt-d07a232a03fa469e8bae47ce2bf9e307

**Comment by tzarebczan** [comment]
- [ ] the drop-down carrot seems too close to the right side of the box
- [ ] This page looks a bit strange to me with some of the grouped settings (tip) and dual pane (Similar to the other dual pane on system settings). Maybe if we added help test under each category and made the Tip heading > "Tipping with Comments" to fill in the heading (+ some simple help text). 
image

- [ ] Bank accounts listed twice

- [ ] Center all pages? This should also say "Add a card" - the heading changes. 
image
image

- [ ] navigation steps count when switching settings pages, so the Back behavior is strange - goes back through each. Back should always go back to the previous non settings page right?

File: static/app-strings.json [hunk 1]
```diff
@@ -1744,8 +1746,9 @@
   "Delegation": "Delegation",
   "Add moderator": "Add moderator",
   "Enter a @username or URL": "Enter a @username or URL",
-  "examples: @channel, @channel#3, https://odysee.com/@Odysee:8, lbry://@Odysee#8": "examples: @channel, @channel#3, https://odysee.com/@Odysee:8, lbry://@Odysee#8",
+  "Enter a channel name or URL to add as a moderator.\nExamples:\n - @channel\n - @channel#3\n - https://odysee.com/@Odysee:8\n - lbry://@Odysee#8": "Enter a channel name or URL to add as a moderator.\nExamples:\n - @channel\n - @channel#3\n - https://odysee.com/@Odysee:8\n - lbry://@Odysee#8",
```
[the remark this answers is not in the document]
Comment (infinite-persistence) [reply static/app-strings.json:1749]: Technically supported in Transifex
image

**Comment by infinite-persistence** [comment]
### 2b53c74 to fc4de99 
- Changed the "Back" button behavior from link-style (ala Chrome) to stack style.
- Removed double "Bank Accounts" (merge error).
- Moved "Homepages" to the top in [Appearance].
- Switched from multi-pane to mult-row style per Tom's feedback.
    - The multi-pane version aimed to provide visual separation between label and value (left is always label, right is always value), but agree that it could be too clamped for radio buttons.
    - 
- Added extra top-margin when navigating to a sub-section.

**Pull request merged**